src/User/components/Aside/Aside.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
-import Logout from "../../components/Buttons/Logout";
```

Comment:
ise Q hta diya ?

Reply:
because I have directly added code of button in Aside.jsx

Reply:
to add sweetAlert